src/main/java/org/opentripplanner/middleware/triptracker/TripInstruction.java

```diff
@@ -121,4 +159,39 @@ private String buildOnTrackInstruction() {
         }
         return NO_INSTRUCTION;
     }
+
+    /**
+     * Build wait for bus instruction.
+     */
+    private String buildWaitForBusInstruction() {
+        String routeShortName = getRouteShortNameFromLeg(busLeg);
+        long delayInMinutes = busLeg.departureDelay;
+        long absoluteMinutes = Math.abs(delayInMinutes);
+        long waitInMinutes = Duration
+            .between(currentTime.atZone(DateTimeUtils.getOtpZoneId()), busLeg.getScheduledStartTime())
+            .toMinutes();
+        String delayInfo = (delayInMinutes > 0) ? "late" : "early";
+        String arrivalInfo = (absoluteMinutes <= 1)
+            ? ", on time"
+            : String.format("now %s %s", getReadableMinutes(delayInMinutes), delayInfo);
+        return String.format(
+            "Wait %s for your bus, route %s, scheduled at %s %s",
```

Comment:
Remove space between the last two `%s`. Right now, if you are late, it prints
```
"Wait  for your bus, route 40, scheduled at 4:14 PM , on time"
```

Reply:
Nit: Transfer the space after "Wait" to before the number of minutes, or make it conditional to minutes being shown.

Reply:
I have addressed the incorrect spaces.